code/style.css

```diff
@@ -1,4 +1,309 @@
-html {
-  background: #0025ff;
-  color: #fff;
+*{
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  list-style: none;
+  box-sizing: border-box;
 }
+
+
+body{
+  width: 100%;
+  min-height: 100%;
+  background-color: #dadada;
+  font-family: Raleway;
+  display: flex;
+  flex-direction: column;
+}
+
+img {
+  width: 90%;
+  height: 90%; 
+}
+
+header{
+display: flex;
+align-items: center;
+width: 100%;
+} 
+
+nav{
+width: 100vw;  
+height: 100px;
+}
+
+.logo{
+  font-family: 'Long Cang';
+  font-size: 60px;
+  color: black;
+  padding: 20px 0px 20px 60px;
+}
+
+.menu-button {
+  display: flex;
+  height: 30px;
+  width: 30px;
+  font-size: 20px;
+  justify-content: flex-end;
+  margin: 10px;
+}
+
+/*removes button styling*/
+button, input[type="submit"], input[type="reset"] {
+	background: none;
+	color: inherit;
+	border: none;
+	padding: 0;
+	font: inherit;
+	cursor: pointer;
+	outline: inherit;
+}
+
+.dropdown {
+  display: flex;
+  justify-content: flex-end;
+  align-content: flex-start; 
```

Comment:
Just by clicking around in the inspector - this doesn't do anything, or does it? ^^